src/Tabs.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import ClassNames from 'classnames';
+import PropTypes from 'prop-types';
+
+export class Tabs extends React.Component {
+  state = {
+    selectedTab: 'Tab 1',
```

Comment:
```suggestion
    selectedTabTitle: 'Tab 1',
```
but better keep selected Tab ID here instead of the title, as IDs are always unique